deploy/deployctl/config.py

```diff
@@ -14,7 +14,8 @@ class Configuration:
     domain: str = None
     environment_tag: str = None
     authorized_networks: str = "0.0.0.0/0"
-    data_pipeline_output: str = None
+    data_pipeline_output: str = "gs://gnomad-browser-data-pipeline"
```

Comment:
Can we leave the defaulting to `None`, since most people can't write to our bucket?  Looks like we check to see if the configuration is complete here: https://github.com/broadinstitute/gnomad-browser/blob/007388ffe8568940f71720bbde20874f9c1bd9ce/deploy/deployctl/subcommands/data_pipeline.py#L24.

You could add another check for the snapshots bucket under that.